pom.xml

```diff
@@ -50,6 +51,7 @@
         <converter.version>0.2.2</converter.version>
         <jline.version>2.14.3</jline.version>
         <joda-converter.version>1.7</joda-converter.version>
+        <curator-framework.version>2.9.0</curator-framework.version>
```

Comment:
I think that is better use the last version of the library (curator-framework 3.3.0).